docs/rules/prefer-arrow-callback.md

```diff
@@ -1,15 +1,22 @@
-# Suggest using arrow functions as callbacks. (prefer-arrow-callback)
+# Suggest using ES6 arrow functions to describe callbacks (prefer-arrow-callback)
 
-Arrow functions are suited to callbacks, because:
+Arrow functions can be an attractive alternative to function expressions, when describing callbacks or function arguments.
```

Comment:
This is very nitpicky, but I think a comma is unnecessary here: `function expressions, when`